### PR TITLE
Return exit code

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

**Comment by jayvdb** [comment]
use case : https://gerrit.wikimedia.org/r/#/c/294907/

**Comment by drmikehenry** [comment]
This is a good idea; thanks, John!  I'd like to give a little more thought to how best to consolidate the exit status from ``find``, ``xargs``, and ``findx`` itself, and make sure all of the exit paths from ``findx`` are covered (due to exceptions, pressing ``Ctrl-c``, etc.), but then I'm planning to add this feature.

File: findx.py [hunk 1]
```diff
@@ -758,7 +762,7 @@ def main():
     f = Findx()
     try:
         f.parse_command_line(sys.argv[1:])
-        f.run()
+        return f.run()
     except FindxError as e:
         print(e)
```
Comment (jayvdb) [inline findx.py]: this should also return a non-zero code.

File: findx.py [hunk 2]
```diff
@@ -758,7 +762,7 @@ def main():
...
     except KeyboardInterrupt as e:
```
Comment (jayvdb) [inline findx.py]: this should also return a non-zero code.

**Comment by jayvdb** [comment]
Nod.  Thanks for taking the time to look for a complete solution.
I've noted the two spots where I can see there is additional need for an exit code.  have i missed any?

**Comment by drmikehenry** [comment]
I've released v0.9.9 with support for returning exit status codes.  See ``findx -help`` for details.

**Comment by jayvdb** [comment]
Thank you! That is a little more complete than I had in mind.  Tested and works well for my needs of course.

**Pull request closed without merging**